File: Java/Mahasiswa.java
public class Mahasiswa extends Civitas {

    private int nim;
    private String fakultas;
    private String prodi;

    public Mahasiswa(int nim, String fakultas, String prodi) {
        this.nim = nim;
        this.prodi = prodi;
    }

    public Mahasiswa() {
    }

    public int getNim() {
        return this.nim;
    }

    public void setNim(int value) {
        this.nim = value;
    }

    public String getFakultas() {
        return this.fakultas;
    }

    public void setFakultas(String value) {
        this.fakultas = value;
    }

    public String getProdi() {
        return this.prodi;
    }

    public void setProdi(String value) {
        this.prodi = value;
    }
}

File: PHP/Mahasiswa.php
<?php

class Mahasiswa extends Civitas
{
    // atribut private
    private $nim, $studi, $fakultas;

    // construct
    public function __construct()
    {
        $this->nim = "";
        $this->studi = "";
        $this->fakultas = "";
    }

    //setter
    public function setNim($nim)
    {
        $this->nim = $nim;
    }
    public function setStudi($studi)
    {
        $this->studi = $studi;
    }
    public function setFakultas($fakultas)
    {
        $this->fakultas = $fakultas;
    }

    //getter
    public function getNim()
    {
        return $this->nim;
    }
    public function getStudi()
    {
        return $this->studi;
    }
    public function getFakultas()
    {
        return $this->fakultas;
    }

    //print
    function printMhs()
    {
        echo "Data Mahasiswa :" . $this->getNim() . " --- " . $this->getFakultas() . " --- " . $this->getStudi() . "<br/>";
    }

    // destruct
    public function __destruct()
    {
    }
}
